app/controllers/operations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class OperationsController < ApplicationController
+  def index
+    @operations = Operation.all
```

Comment:
Let's consider pre-loading here - see the comment below.

Reply:
Done! I think you need both the `includes` and `joins` for this, but I've been known to be wrong. Once or twice. Per hour. Ok fine. Per half-hour.